device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-C224O8/pg_profile_lookup.ini

```diff
@@ -0,0 +1,50 @@
+##
+## Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+## Apache-2.0
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+## http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##
+#  PG lossless profiles.
+# speed cable  size   xon    xoff   threshold
+  10000    5m 19456   19456  20480         0
+  25000    5m 19456   19456  21504         0
+  40000    5m 19456   19456  24576         0
+  50000    5m 19456   19456  26624         0
+ 100000    5m 19456   19456  44032         0
+ 200000    5m 19456   19456  50176         0
+ 400000    5m 19456  19456  86016         0
+ 800000    5m 38912  38912  95232         0
+  10000   40m 19456   19456  20480         0
+  25000   40m 19456   19456  22528         0
+  40000   40m 19456   19456  25600         0
+  50000   40m 19456   19456  28672         0
+ 100000   40m 19456   19456  49152         0
+ 200000   40m 19456   19456  60416         0
+ 400000   40m 19456  19456  104448        0
+ 800000   40m 38912  38912  132096        0
+  10000  300m 19456   19456  15360         0
+  25000  300m 19456   19456  22528         0
+  40000  300m 19456   19456  31744         0
+  50000  300m 19456   19456  37888         0
+ 100000  300m 19456  19456  74752         0
+ 200000  300m 19456  19456  120832        0
+ 400000  300m 19456  19456  235520        0
```

Comment:
xoff 300m 400G 243712?
according to the document the downlink cable length is 250m, so let's use it. xoff 250m 100G 76800

Reply:
The Arch excel has a cable length drop-down option for the calculation.
I only saw 300m so used the closest one.
I'll try to use 250m

Reply:
Updated the file to 250m